starter_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# The starter script for each VM instance of VMN
+# It installs the VNCServer, sets its password, starts a server, and
+# initiates noVNC's Websocket to TCP proxy/bridge Websockify.
+
+# This URL accesses the metadata value of key "session-id" of the current
+# VM, it will be used as a password to connect to the VNCServer.
+PASSWRD=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/session-id -H "Metadata-Flavor: Google")
+export USER=/root
+apt-get update
+sudo apt install -y xfce4 xfce4-goodies tightvncserver
+git clone https://github.com/novnc/noVNC.git
+mkdir ~/.vnc
+echo "${PASSWRD}" | vncpasswd -f > ~/.vnc/passwd
+chmod 600 ~/.vnc/passwd
+HOME=~ vncserver
+cd noVNC || exit
```

Comment:
If your shell script relies on being in a certain working directory, make sure to verify that somewhere. Were do you expect this to run?

Reply:
its important for the VM to be in noVNC for the execution of the last line, i added an exit just in case the cd was unsuccessful. if there's no exit then the current working directory would be noVNC, which is what we want